CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 #### Data migrations
 - [#1126](https://github.com/LayerManager/layman/issues/1126) Migrate graphic URLs in map metadata from workspace&name-based format to UUID-based format.
 - [#1126](https://github.com/LayerManager/layman/issues/1126) Migrate graphic URLs in layer metadata from workspace&name-based format to UUID-based format.
+- [#1126](https://github.com/LayerManager/layman/issues/1126) Migrate map file endpoint URLs in map metadata from workspace&name-based format to UUID-based format.
```

Comment:
I like more to join this item with the "Migrate graphic URLs in map metadata from workspace&name-based format to UUID-based format." as it is one migration.